src/include/planner/plannodes/insert_plan_node.h

```diff
@@ -301,22 +217,19 @@ class InsertPlanNode : public AbstractPlanNode {
   catalog::table_oid_t table_oid_;
 
   /**
-   * values to insert
-   */
-  std::vector<type::TransientValue> values_;
-
-  /**
-   * parameter information
+   * vector of values to insert. Multiple vector of values corresponds to a bulk insert. Values for each tuple are
+   * ordered the same across tuples. Parameter info provides column mapping of values
    */
-  std::vector<ParameterInfo> parameter_info_;
+  std::vector<std::vector<type::TransientValue>> values_;
 
   /**
-   * name of time to insert
+   * parameter information. Provides which column a value should be inserted into. For example, for a tuple t at
```

Comment:
What are best practices for documenting a member and also its accessor? It feels a bit redundant to have this here again.